pages/index.tsx

```diff
@@ -1,5 +1,135 @@
+import { makeStyles } from "@mui/styles";
+import Image from "next/image";
+
+import { Total } from "../components/stats/Total";
+import { Classes } from "../components/stats/Classes";
+import { Destinations } from "../components/stats/Destinations";
+import { Authors } from "../components/stats/Authors";
+
+type StatsResponse = Record<string, string>;
+
+async function fetcher<JSON = any>(input: RequestInfo, init?: RequestInit): Promise<JSON> {
+  const res = await fetch(input, init);
+  return res.json();
+}
+
+const useStyles = makeStyles({
+  grid: {
+    padding: "1rem",
+    minHeight: "100vh",
+    display: "grid",
+    gridTemplateColumns: "repeat(12, 1fr)",
+    gridTemplateRows: "1fr 1fr 2fr",
+    gridGap: "1rem",
+    backgroundColor: "#eeeeee",
+  },
+});
+
 const Home = () => {
-  return <div>helo</div>;
+  const { data, error } = useSWR<StatsResponse>("/api/stats", fetcher, { refreshInterval: 5000 });
+  const classes = useStyles();
+  const mobileLayout = useMediaQuery("(max-width:1250px)");
+
+  if (error) return <div>failed to load</div>;
+  if (!data) return <div>loading...</div>;
+
+  return (
+    <Box
+      className={classes.grid}
+      style={
+        mobileLayout ? { gridTemplateColumns: "auto", gridTemplateRows: "repeat(5, auto)" } : {}
+      }
+    >
+      <Box
+        style={{ gridRow: mobileLayout ? "1" : "1 / 3", gridColumn: mobileLayout ? "1" : "1 / 4" }}
+      >
+        <Paper
+          elevation={12}
+          sx={{
+            p: 3,
+            width: "100%",
+            height: "100%",
+            display: "flex",
+            flexDirection: "column",
+            justifyContent: "center",
+            alignItems: "center",
+          }}
+        >
+          <Image
+            src="/logo.png"
+            alt="Amnesty International Logo"
+            width={mobileLayout ? 200 : 400}
+            height={mobileLayout ? 200 : 400}
+          />
+          <h1
+            style={{
+              fontFamily: "Amnesty Trade Gothic Bold Condensed",
+              margin: "0",
+              textTransform: "uppercase",
+              fontSize: 36,
+              textAlign: "center",
+            }}
+          >
+            Amnesty International ZSK
+          </h1>
+        </Paper>
+      </Box>
+      <Box style={{ gridRow: mobileLayout ? "2" : "1", gridColumn: mobileLayout ? "1" : "4 / 10" }}>
+        <Total
+          totalLetters={data.totalLetters}
+          totalDestinations={data.totalDestinations}
+          totalAuthors={data.totalAuthors}
+        />
+      </Box>
+      <Box style={{ gridRow: mobileLayout ? "3" : "2", gridColumn: mobileLayout ? "1" : "4 / 10" }}>
+        <Authors
+          authors={Object.keys(data)
+            .filter((key) => key.startsWith("top."))
+            .map((key) => {
+              return {
+                name: `${key.split(".")[2]} ${key.split(".")[3]}`,
+                value: parseInt(data[key]),
+              };
+            })}
```

Comment:
Extract that to a separate function (or two).

First one would be just for extracting the desired keys (for example `extractFilters(data, "top.")`). And the second one for transforming the data, to fit the component's props.